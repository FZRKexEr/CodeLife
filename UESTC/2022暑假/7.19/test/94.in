10
alkztegbdyecixsstqrexiwitixnmuczgpvbzwztjphqozimiivcryetlnctzorzqssnszadvufayxjqhgrpptosadyfvteczaot
10
fcdgkujfllldgnoqusutrujsljivvqzejanihzkrsdgtugdvssahybtlnzwfkhavbiignoxmpdxmlfxhnynqwygmlwjaiamjwlrr
7
ggjiwrgemprfhtilktcccbczhekymnwotddxjvgnsirtuuggtqnxoadfwsvxkylvzydiiztxqlwfdsewsdnffdwlkxtziarrmhdb
3
noakjruxcsfkoqtfvqjckbcpqrwdewncrthfxsvokkmyrtagupskdhefscmebkzsgrxoquxwgxxvhxxlwferbwbucvkvvaeaihxa
8
uyhqfxpbjfmvvpdwffuwczijzzeiechqtusbpzycwpkojscdramolbktigpowmwosmhtastwiqshjawcwrbiypczloweniuttvds
1
ihgwocaplbzrngiihuksnyqbkznccczvkqolggwzrvchkcdsmjpultuxcsbiugtnmeuujaftikfykahsgkilsnilkllqphmlyiey
5
akxhjfeqskkpmbgteqnelzzhnbhctdqohqzadugbfcbuxpnbbghtvrvqrtmntdayrvtnmetzjevtyodtsuvdlgajbdvupnofuapl
2
zrxunoteuisditsvxfnvpxymujwwfraqtevgpnajqaivupymitoxaviulceukolgwtfddggglugthwwvznkgtqxvdhclhgzrzfaf
9
siweboqxgwujvwwzcgtxeywqqlsdhxyquyoqxmwxzzhpfjqtworxmuvcotqoxfmqiksguicgdynocraltcngaxosvcqjvvfwxafy
8
hvcbtmunjowsbjvuojrmfcltyastfjpupcshmsrdgsqwwthgycnzrmjhglbsnjndovkbgfjvkiiualkpviyvrawgpepciktijosj
3
