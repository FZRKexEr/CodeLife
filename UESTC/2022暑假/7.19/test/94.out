1024
16384
603979768
4096
882499718
1048576
65980984
2048
4096
603979768
